spacewalk/config/spacewalk-config.spec

```diff
@@ -105,6 +111,7 @@ ln -sf  %{apacheconfdir}/conf/ssl.crt/server.crt $RPM_BUILD_ROOT/etc/pki/tls/cer
 %defattr(-,root,root,-)
 %attr(400,root,root) %config(noreplace) %{_sysconfdir}/rhn/spacewalk-repo-sync/uln.conf
 %config(noreplace) %{apacheconfdir}/conf.d/zz-spacewalk-www.conf
+%config(noreplace) %{apacheconfdir}/conf.d/zz-spacewalk-docs.conf
```

Comment:
Why not ship this file with the doc package?

Reply:
@juliogonzalez and I chose spacewalk-config because it was simpler (only git work) and all the other apache config is here.
On the other hand, with susemanager-docs, I need to add the config file to git, edit the Makefile to add the file to the tarball and submit the specfile on OBS.
To me, both options would be ok. If you have any preference for the docs package, I can still move it there.